src/tree/ResolvableTreeItemBase.ts

```diff
@@ -53,7 +61,10 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
 
             const resolver = this.getResolver();
 
-            await this.runWithTemporaryDescription(context, 'Loading...', async () => {
+            await this.runWithTemporaryDescription(context, {
+                description: loading,
+                skipRefresh: true
+            }, async () => {
                 this.resolveResult = await resolver.resolveResource(this.subscription, this.data);
 
                 // Debug only?
```

Comment:
There's a comment below (but GitHub won't let me add it in context):
> `// It is not needed to refresh at this point, because runWithTemporaryDescription already does that`

Can we make sure that resolving can update the label/description? I _think_ so but I'm not totally sure. Stopping a function app makes resolving update the description so IMO that's a sufficient test--if that "Stopped" description still appears I'm happy.

Reply:
:chef's kiss:
Perfect! Also, love how fast SWA resolves.